lib/page-object/accessors.rb

```diff
@@ -1220,8 +1220,21 @@ def svg(name, identifier={:index => 0}, &block)
     # @param optional block to be invoked when element method is called
     #
     def element(name, tag, identifier={:index => 0}, &block)
+      if tag.is_a? Hash
+        identifier = tag
```

Comment:
Oh nice, I didn't notice your PR before. Yes, it should be applied to def elements as well.